substrate/frame/revive/src/vm/mod.rs

```diff
@@ -47,12 +47,12 @@ use frame_support::{
 use sp_core::{Get, H256, U256};
 use sp_runtime::DispatchError;
 
-/// Validated Wasm module ready for execution.
+/// Validated Vm module ready for execution.
 /// This data structure is immutable once created and stored.
 #[derive(Encode, Decode, scale_info::TypeInfo)]
 #[codec(mel_bound())]
 #[scale_info(skip_type_params(T))]
-pub struct WasmBlob<T: Config> {
+pub struct VmBlob<T: Config> {
```

Comment:
Same here (and in other places too) - I am not sure if the generic term is appropriate everywhere. Reading `VmBlob` elsewhere in the code base is a bit confusing, I think `PvmBlob` or `ContractBlob` would be better.